server/service/src/dashboard/stock_expiry_count.rs

```diff
@@ -35,9 +35,92 @@ impl StockExpiryCountServiceTrait for StockExpiryServiceCount {
                         before_or_equal_to: Some(date_time),
                         after_or_equal_to: None,
                     })
-                    .store_id(EqualFilter::equal_to(&store_id)),
+                    .store_id(EqualFilter::equal_to(&store_id))
+                    .is_available(true),
```

Comment:
Assume we only want to see if the available packs is expired since other packs would have been allocated already?

Reply:
the name mislead me - I was going to say that we don't want `is_available` but instead we want to check if there is some stock, i.e. `available_number_of_packs` > 0

turns out, that's what `is_available` does. so, thanks. that's all good.